README.md

```diff
@@ -38,6 +38,8 @@ To install the latest versions of all of the Orquestra Core packages together ru
 
 `pip install orquestra-core`
 
+Libraries to interact with quantum simulators or QPU needs to be installed seperately. i.e `pip install orquestra[cirq]`.
```

Comment:
I'd rephrase it to sth like:
"Libraries to interact with other frameworks (e.g. qiskit, cirq), simulators or QPU can be installed as optional dependencies, i.e.: `pip install orquestra[cirq]`"